### PR TITLE
Add AB test for Public Good ad vendor

### PR DESCRIPTION
**Pull request by Jakeii**
## What does this change?
This will add a 0% AB test for the new ad format.

## Why?
We have a new vendor called "Public Good" which serves via a custom creative in GAM so that we can keep their third-party scripts in a safeframe.

The vendor will show in a new slot at the end of articles when the Epic doesn't appear.

## Screenshots
Screenshot 2023-07-31 at 13 11 18

### CODE REVIEW

File: dotcom-rendering/src/components/SlotBodyEnd.importable.tsx [hunk 1]
```diff
@@ -194,6 +209,16 @@ export const SlotBodyEnd = ({
 				<SelectedEpic />
 			</div>
 		);
+	} else if (showArticleEndSlot) {
+		return (
+			<div id="slot-body-end">
+				<AdSlot
+					data-print-layout="hide"
+					position="article-end"
+					display={articleDisplay}
```
Comment (domlander) [inline dotcom-rendering/src/components/SlotBodyEnd.importable.tsx:218]: I don't think you need to pass in `display`. Only the `right` ad slot depends on this.

**Comment by Jakeii** [comment]
> Are you collecting metrics for this test?

Good point, will be when this goes >0%, I'll add CWV collection :)

**Comment by domlander** [comment]
The ad looks much better in that slot than it did in `merchandising-high`!

File: dotcom-rendering/src/components/AdSlot.tsx [hunk 2]
```diff
@@ -642,6 +642,32 @@ export const AdSlot = ({
...
+						className={[
+							'js-ad-slot',
+							'ad-slot',
+							'ad-slot--article-end',
+							'ad-slot--rendered',
+						].join(' ')}
+						css={[
+							css`
+								position: relative;
+								min-height: 450px;
```
[the remark this answers is not in the document]
Comment (Jakeii) [reply dotcom-rendering/src/components/AdSlot.tsx:659]: Good catch, actually, even if I move it to the bottom of the array it still flickers back to 250px when the fluid class is added 😬

File: dotcom-rendering/src/components/SlotBodyEnd.importable.tsx (hunk 1 above)
Comment (Jakeii) [reply dotcom-rendering/src/components/SlotBodyEnd.importable.tsx]: I will raise in a separate PR though I think

**Pull request merged**